pom.xml

```diff
@@ -143,7 +143,7 @@
     <parquet.version>1.13.1</parquet.version>
     <orc.version>1.9.2</orc.version>
     <orc.classifier>shaded-protobuf</orc.classifier>
-    <jetty.version>9.4.53.v20231009</jetty.version>
+    <jetty.version>10.0.18</jetty.version>
```

Comment:
Really minor: let's go to 10.0.19: https://github.com/jetty/jetty.project/releases/tag/jetty-10.0.19

Reply:
Done!